mu-plugins/blocks/horizontal-slider/src/block.js

```diff
@@ -95,10 +95,12 @@ function Block( { items, title } ) {
 	const height = width * aspectRatio;
 
 	return (
-		<div>
+		<div aria-labelledby="sliderHeading">
 			<div className="horizontal-slider-header">
 				<span>
```

Comment:
Is this tag necessary? Seem like the h3 can be a block level element, and there aren't any styles targeting the span as far as I can see

![Screen Shot 2022-10-17 at 7 45 42 PM](https://user-images.githubusercontent.com/1017872/196107299-c2cb6935-a314-432d-946e-6e8e263de1a1.jpg)